packages/node_modules/@webex/plugin-meetings/test/unit/spec/testUtils.js

```diff
@@ -0,0 +1,110 @@
+
+/**
+   * @param {number} max
+   * @param {string} event
+   * @param {EventEmitter} emitter
+   * @param {string} msg
+   * @returns {Promise<mixed>} Resolves with the results of the event
+   */
+
+const max = 20000;
+const waitForSpy = (spy, event) => {
+  let timer;
+  return Promise.race([
+    new Promise((resolve, reject) => {
+      timer = setTimeout(() => {
+        reject(new Error(` ${event} did not fire`));
+      }, max);
+    }),
+    new Promise((resolve) => {
+      const interval = setInterval(() => {
+        if (spy.calledOnce) {
+          clearTimeout(timer);
+          clearInterval(interval);
+          resolve(spy.args[0][0]);
+        }
+      }, 1000);
+    })
+  ]);
+};
+
+const waitForStateChange = (meeting, state) => {
```

Comment:
waitForMeetingStateChange -- how will this be affected by the meeting state machine updates that are coming up?